modal_proto/api.proto

```diff
@@ -90,6 +97,68 @@ enum WebhookAsyncMode {
   WEBHOOK_ASYNC_MODE_AUTO = 4; // redirect to polling endpoint if execution time nears the http timeout
 }
 
+// A web endpoint connection-related message.
```

Comment:
would it make sense to put this in a separate proto file?

Reply:
Hmm I don't have strong opinions either way. I think it's possible, but also everything is contained in a single message right now.